src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -433,4 +435,11 @@ protected function loadSession(array &$session = null)
         $this->started = true;
         $this->closed = false;
     }
+
+    final protected function checkSessionExtensionLoaded()
```

Comment:
I'd suggest removing the method and inlining the logic - I'm fine with the repetition personally.

Reply:
Aggree. Done